### ISSUE TITLE
Summary element in the Basic Accordion demo does not have a focus ring

### ISSUE THREAD

**Issue by foxbunny** [Pure-Web-Foundation/pure-ds#5]
## Reproduction

1. Open the Basic Accordion demo
2. Click above the first accordion
3. Press Tab key
4. Observe no change in appearance

Minimal reproduction (repo, codesandbox, or snippet):

Default storybook will work

## Expected behavior

See a focus ring around the `<summary>` element.

## Screenshots / logs

<img width="729" height="425" alt="Image" src="https://github.com/user-attachments/assets/39487f5b-f825-4bd1-abef-e8e1b6da9305" />

## Environment
- Browser: Brave/Chromium 142.0.7444.176
- Bundler (if any):
- OS: Mac OS Tahoe
- PDS version: main
